roles/contiv/tasks/default_network.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Is there a reason shell is used here instead of command?
```

Comment:
No, I think command should work for all of the tasks here. Shell is only necessary when you expect shell variable evaluation or pipes from one command to another.